src/UCTSearch.cpp

```diff
@@ -235,6 +235,39 @@ void UCTSearch::dump_stats(FastState & state, UCTNode & parent) {
     tree_stats(parent);
 }
 
+void UCTSearch::output_analysis(FastState & state, UCTNode & parent) {
+    if (!parent.has_children()) {
+        return;
+    }
+
+    const int color = state.get_to_move();
+
+    // sort children, put best move on top
+    parent.sort_children(color);
+
+    if (parent.get_first_child()->first_visit()) {
+        return;
+    }
+
+    int movecount = 0;
+    std::string separator = "info";
+    for (const auto& node : parent.get_children()) {
+        // Always display at least two moves. In the case there is
+        // only one move searched the user could get an idea why.
+        if (++movecount > 2 && !node->get_visits()) break;
```

Comment:
If there is a visual analysis GUI I guess this is less important, so the "at least 2" part can go.

Reply:
Ok, agree.